### DISCUSSION TITLE
Adding input within a callout

### DISCUSSION THREAD

**Discussion by stuarttravers** [mProjectsCode/obsidian-meta-bind-plugin#377] [General]
I use a "INPUT[textArea:summary]" command with most of my notes to have a quick summary of my notes so that when they show up in dataview queries, I get the summary. I routinely have the query wrapped in a callout but since the latest obsidian update 1.6.7, the input in live preview isn't working. It will work in reading mode and if the code is not in a callout. Has anyone else seen this behaviour? Instead of allowing text input, clicking in the text area shows the source code.

**Comment by bpannier** [comment]
Indeed, I have the same problem - this is very unfortunate as this removes the ability to use the plugin and have a page organised.

How to reproduce:

```
> [!important]+
> **🙌😃 Highlights of the week** 
> `INPUT[textArea(placeholder(Successes; Good feelings; Happyiness)):habit.grateful]`
>
```

I also played around with how to avoid this behavior, like using the Columns plugin, which removes this behavior but creates completely new issues with tasks...

**Reply by mProjectsCode** [reply]
As stated in #403, I had a fix that broke other things. I am open to a PR if someone finds a fix that doesn't break anything else.

**Reply by LynetteCullens** [reply]
> As stated in #403, I had a fix that broke other things. I am open to a PR if someone finds a fix that doesn't break anything else.

What language would the fix be written in? Typescript?

**Reply by LynetteCullens** [reply]
> Indeed, I have the same problem - this is very unfortunate as this removes the ability to use the plugin and have a page organised.
> 
> 
> 
> How to reproduce:
> 
> 
> 
> ```
> 
> > [!important]+
> 
> > **🙌😃 Highlights of the week** 
> 
> > `INPUT[textArea(placeholder(Successes; Good feelings; Happyiness)):habit.grateful]`
> 
> >
> 
> ```
> 
> 
> 
> I also played around with how to avoid this behavior, like using the Columns plugin, which removes this behavior but creates completely new issues with tasks... 
> 
> 

I have a JavaScript code that runs automatically using RunJS every time my vault opens a new note. I haven't checked if it still works in like 5 updates though. You can find it in the issues section.

**Reply by mProjectsCode** [reply]
> > As stated in #403, I had a fix that broke other things. I am open to a PR if someone finds a fix that doesn't break anything else.
> 
> What language would the fix be written in? Typescript?

Yes, as part of the plugin code.
